lib/liquid/nimble_parser.ex

```diff
@@ -88,6 +91,12 @@ defmodule Liquid.NimbleParser do
 
   defparsec(:include, Include.tag())
 
+  defparsec(:body_elsif, If.body_elsif())
+  defparsec(:body_if, If.body())
```

Comment:
`body_elsif` and `elsif_tag` are needed because they are recursively called each other. Use private functions here provocates a infinite recursion in compile time. `body_if` can be deleted. Corrected.